Subject: [PATCH] Local m68k changes

Local m68k changes, _NEVER_ to be submitted upstream:
  - Force ARCH to be m68k
  - Append `-m68k' to EXTRAVERSION

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org> for m68k CVS only
---
 Makefile          |    2 +-
 localversion.m68k |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -190,7 +190,7 @@ SUBARCH := $(shell uname -m | sed -e s/i
 # Default value for CROSS_COMPILE is not to prefix executables
 # Note: Some architectures assign CROSS_COMPILE in their arch/*/Makefile
 export KBUILD_BUILDHOST := $(SUBARCH)
-ARCH		?= $(SUBARCH)
+ARCH		?= m68k
 CROSS_COMPILE	?=
 
 # Architecture as present in compile.h
--- /dev/null
+++ b/localversion.m68k
@@ -0,0 +1 @@
+-m68k
